app/facades/food_facade.rb

```diff
@@ -0,0 +1,9 @@
+class FoodFacade
+  def self.food_finder(food_string)
+    foods_data = FoodService.food_search(food_string)
+    
+    foods_data[:foods][0..9].map do |food_data|
```

Comment:
If you add that param for `pageSize=10` in your Service, you would not need to set a range here.